index.ts

```diff
@@ -1,24 +1,34 @@
+
+console.group("Cargando los resultados...");
 //  1.- Create an array of n unique numbers, from 0 to N - 1, where N is the length of the array.
 //  i: 5
 //  o: [0, 1, 2, 3, 4]
+const crearCajaInicial = (n: number): number[] => {
+  return Array.from({ length: n }, (_, x) => x);
+}
+
+const miCajaPrimera: number[] = crearCajaInicial(5);
```

Comment:
The challenge never mencioned nothing about "Boxes" where did you get that ?

Reply:
se eliminaron las variables nombradas como "cajas"